### ISSUE TITLE
Parse already existing .osb files

### ISSUE THREAD

**Issue by Ballasi** [Ballasi/osb#10]
That's something I already did on my old project `storygen`. It could be very interesting to have this implemented out-of-the-box

**Comment by DoomDuck** [comment]
Does this means create a parser for .obs files ?

**Comment by Ballasi** [comment]
Yes, I'm thinking of something that takes a `.osb` file and returns a `Module` that corresponds to the contents of that file.
